config/default.yml

```diff
@@ -2776,6 +2776,12 @@ Style/GuardClause:
   # needs to have to trigger this cop
   MinBodyLength: 1
 
+Style/HashEachMethods:
+  Description: 'Use Hash#each_key and Hash#each_value.'
+  StyleGuide: '#hash-each'
+  Enabled: true
```

Comment:
Please, change this to `pending`. We're no longer enabling any new cops by default.